### PR TITLE
Java 16710

### PR DESCRIPTION
**Pull request by anastasiosioannidis**

### CODE REVIEW

File: core-java-modules/core-java-jndi/src/test/java/com/baeldung/jndi/AJndiUnitTest.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 
-class JndiUnitTest {
+class AJndiUnitTest {
```
Comment (dkapil) [inline core-java-modules/core-java-jndi/src/test/java/com/baeldung/jndi/AJndiUnitTest.java:17]: do we need this rename change?
and the comment below?

**Review by dkapil — CHANGES_REQUESTED** [review]

**Comment by anastasiosioannidis** [comment]
retest this please

File: core-java-modules/core-java-jndi/src/test/java/com/baeldung/jndi/AJndiUnitTest.java (hunk 1 above)
Comment (anastasiosioannidis) [reply core-java-modules/core-java-jndi/src/test/java/com/baeldung/jndi/AJndiUnitTest.java]: I fixed these 2 points and retested the PR successfully.
I guess all is fine, we can merge this.

**Comment by anastasiosioannidis** [comment]
retest this please

**Pull request merged**